python/cudf/cudf/core/column/string.py

```diff
@@ -730,30 +730,42 @@ def wrap(self, width, **kwargs):
             msg = f"width must be of integer type, not {type(width).__name__}"
             raise TypeError(msg)
 
-        if "expand_tabsbool" in kwargs:
-            raise NotImplementedError(
-                "`expand_tabsbool` parameter is not supported"
-            )
+        expand_tabs = kwargs.get("expand_tabs", False)
+        if expand_tabs:
+            raise NotImplementedError("`expand_tabs=True` is not supported")
+        warnings.warn(
+            "wrap current implementation defaults to `expand_tabs`=False"
+        )
```

Comment:
Should follow a similar pattern for other kwargs where we should warn if our default value differs from Pandas and then we should throw if the user explicitly passes a value we don't yet support.
```suggestion
        expand_tabs = kwargs.get("expand_tabs", None)
        if expand_tabs == True:
            raise NotImplementedError("`expand_tabs=True` is not supported")
        elif expand_tabs is None:
            warnings.warn(
                "wrap current implementation defaults to `expand_tabs`=False"
            )
```

Reply:
Made similar changes for the rest of the parameter settings.